dist/angular-patternfly.js

```diff
@@ -2767,6 +2767,345 @@ angular.module('patternfly.views').directive('pfDataList', [
+ * <li>.onDblClick             - ( function(item, event) ) Called to notify when an item is double clicked, default is none
+ * </ul>
+ *
+ * @param {Array} items the data to be shown in the data tiles<br/>
+ *
+ * @example
+ <example module="patternfly.views" deps="patternfly.utils">
+ <file name="index.html">
+   <style>
+   hr {
+          display: block;
+          height: 10px;
+          border: 0;
+          border-top: 1px solid #525252;
+          margin: 1em 0;
+          padding: 0;
+        }
+   </style>
+   <div ng-controller="ViewCtrl" class="row" style="display:inline-block; width: 100%;">
+     <div class="col-md-12">
+       <div pf-data-tiles id="exampleDataTiles" config="config" items="items">
+         <div class="col-md-12">
+           <span>{{item.name}}</span>
+         </div>
+         <div class="col-md-12">
+           <span>{{item.address}}</span>
+         </div>
+         <div class="col-md-12">
+           <span>{{item.city}}, {{item.state}}</span>
+         </div>
+       </div>
+     </div>
+     <hr class="col-md-12">
+     <div class="col-md-12">
+       <form role="form">
+         <div class="form-group">
+           <label>Selection</label>
+           </br>
+           <label class="radio-inline">
+             <input type="radio" ng-model="selectType" value="checkbox" ng-change="updateSelectionType()">Checkbox</input>
+           </label>
+           <label class="radio-inline">
+             <input type="radio" ng-model="selectType" value="tile" ng-change="updateSelectionType()">Tile</input>
+           </label>
+           <label class="radio-inline">
+             <input type="radio" ng-model="selectType" value="none" ng-change="updateSelectionType()">None</input>
+           </label>
+         </div>
+       </form>
+     </div>
+     <div class="col-md-12">
+       <form role="form">
+         <div class="form-group">
+           <label class="checkbox-inline">
+             <input type="checkbox" ng-model="config.dblClick">Double Click</input>
+           </label>
+           <label class="checkbox-inline">
+             <input type="checkbox" ng-model="config.multiSelect" ng-disabled="config.dblClick">Multi Select</input>
+           </label>
+         </div>
+       </form>
+     </div>
+     <div class="col-md-12">
+       <form role="form">
+         <div class="form-group">
+           <label class="checkbox-inline">
+             <input type="checkbox" ng-model="showDisabled">Show Disabled Rows</input>
+           </label>
+         </div>
+       </form>
+     </div>
+     <div class="col-md-12">
+       <label style="font-weight:normal;vertical-align:center;">Tile Width: </label>
+       <input style="height:25px; width:60px;" type="number" ng-model="config.width"></input>
+       <label style="font-weight:normal;vertical-align:center;margin-left:20px;">Tile Height: </label>
+       <input style="height:25px; width:60px;" type="number" ng-model="config.height"></input>
```

Comment:
Why are all of these inline styles necessary?  Can we somehow use the already available styling or put these into a class?